src/sentry/static/sentry/app/views/organizationIncidents/details/body.jsx

```diff
@@ -37,6 +39,21 @@ export default class DetailsBody extends React.Component {
     const {activeTab} = this.state;
     const ActiveComponent = TABS[activeTab].component;
 
+    const detectedTs = incident && moment.utc(incident.dateStarted).unix();
+    let closestTimestampIndex;
+    const chartData =
+      incident &&
+      incident.eventStats.data.map(([ts, val], i) => {
+        if (ts > detectedTs) {
+          closestTimestampIndex = i > 0 ? i - 1 : 0;
+        }
+        return [
+          ts * 1000,
+          val.length ? val.reduce((acc, {count} = {count: 0}) => acc + count, 0) : 0,
+        ];
+      });
+    const markPointCoordinate = chartData && chartData[closestTimestampIndex];
```

Comment:
Sooo I'll have to see how this looks with real data, but echarts requires an explicit (x,y) value to draw the symbol (in this case it's the incident detected bubble). I'm using the closest date (the price is right style). AFAICT we can't give it an x-axis value and have it draw on the line, so we may need to just calculate the y-axis value ourselves.